torch_geometric/nn/models/label_usage.py

```diff
@@ -0,0 +1,120 @@
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+
+from torch_geometric.typing import Adj
+
+
+class LabelUsage(torch.nn.Module):
+    r"""The label usage operator for semi-supervised node classification,
+    as introduced in `"Bag of Tricks for Node Classification"
+    <https://arxiv.org/abs/2103.13355>`_ paper.
+
```

Comment:
I think we should give a brief description of how it works here for folks that are too lazy to read the paper :P